### DISCUSSION TITLE
How to disable "Jira Assist: Start time tracking" ?

### DISCUSSION THREAD

**Discussion by gavar** [shridhar-tl/jira-assistant#233] [Q&A] [answered]
Hi all,

Some time ago I've noticed that a new arrow icon appears on every ticket on the Active Sprints boards. I have gone through all settings in Jira Assistant plugin, but cannot find any way to prevent that from showing up. Is there any way to disable that feature?

Thank you!

![image](https://user-images.githubusercontent.com/3128402/194648377-6dd4139f-ceef-4a7d-b53c-3565c2537f1c.png)

**Comment by shridhar-tl** [comment, marked as answer]
Hi @gavar - This feature was implemented and published in Aug 2022 as part of Jira Assistant v2.41. When you say it started appearing some time back, it means to me that you have been using old version of Jira Assistant since then and recently it got updated.

Additionally to try new features, you can switch to Web version which could be unstable sometimes, but would receive quick fixes and updates/enhancements. You would be able to switch back to extension any time just with couple of clicks.

Anyway, to answer your question, yes, it is possible to disable this option. Go to General settings -> Time tracker tab and unselect the following settings: "Attach tracker functionality within Jira"

![image](https://user-images.githubusercontent.com/37339683/194685218-46b9b878-e57b-4822-b400-d6d4f618873e.png)

Have a look at #222 for more details or provide your feedback.

**Reply by gavar** [reply]
Omg, how can I miss that section, thank you a lot! 

I don't have any particular feedback as I'm using [TopTracker](https://www.toptal.com/tracker) for time tracking but thank you a lot for keeping the product alive, I use that every day to fill my timesheets and see a summary of those!

**Comment by shridhar-tl** [comment]
This issue has been resolved long back and hence closing this ticket. Please feel free to reopen it if you have any other queries.
